app/Http/Controllers/AttendanceController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Models\Attendance;
+use Illuminate\Http\Request;
+
+class AttendanceController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('attendance.index');
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('attendance.create');
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function register()
+    {
+        return view('attendance.register');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
```

Comment:
You will need to use requests for forms. This is to prevent things like a hacker corrupting our db. You can find a guide to this here: https://imali.frog.co.ke/form-requests-validation-in-laravel/